source/manual/rabbitmq.html.md

```diff
@@ -4,147 +4,123 @@ title: Manage RabbitMQ
-
-## Inspecting/removing items from a queue
-
-We had an instance where an application was unable to process a message
-in the queue, but left the message on the queue. This meant it was
-backing up. Removing the message from the queue was the right solution
-in our case.
-
-1. Find the queue
-
-   Click on the "Queues" tab. Then click on the name of the queue.
-
-2. Find the messages
-
-   Scroll down and click "Get messages". Clicking the "Get Message(s)"
-   button that appears will fetch however many messages you ask for.
-
-   > **Note**
-   >
-   > Fetching messages actually removes them from the queue. By leaving
-   > the "Requeue" option set to "Yes", they will be added back to queue.
-
-3. Delete the messages
-
-   > **Note**
-   >
-   > There is a risk that you might delete the wrong message(s). This
-   > is because the contents of the queue may have changed.
-
-   Repeat, but change the "Requeue" option to "No".
-
-## Previewing a message for a document_type
-
-The publishing API generates messages when content is updated and posts them
-to the rabbitMQ exchange. Each message has a shared format, however the contents
-of the message is affected by the publishing app and what data it sends to the
-publishing API.
+## How we run RabbitMQ
 
-As messages for different formats can vary, we have created a rake task in the
-publishing API app to allow us to easily generate example messages. The example
-message is generated from the most recently published message (based off of
-last public_updated_at) for the entered document type:
+### Overview
+
+![A graph showing the message flow](images/rabbitmq_graph.png)
+
+**Producer**: an application that publishes messages RabbitMQ. On GOV.UK this could
+be [publishing-api](https://github.com/alphagov/publishing-api).
+
+**Exchanges** are AMQP entities where messages are sent. They take a message
+and route it into zero or more queues. The routing algorithm used depends on
+the exchange type and rules called _bindings_.  When a content change is made
+in a publishing application (e.g. Travel Advice Publisher), Publishing API
+[publishes a message][publishing_api_publishes_message] to our main exchange,
+`publishing_documents`.
+
+**Queues** in the AMQP model queues are very similar to queues in other message
+and task-queueing systems: they store messages that are consumed by
+applications. An example of a queue is `email_alert_service` which is used by
+[email-alert-service][email-alert-service] to forward publishing activity to
+email-alert-api. Queues are [created by consumer
+applications][create_queues].
+
+**Bindings** are rules that exchanges use (among other things) to route
+messages to queues. To instruct an exchange E to route messages to a queue Q, Q
+has to be bound to E. Bindings may have an optional routing key attribute. An
+example of a binding is the `cache_clearing_service-high` queue is bound to the
```

Comment:
Minor: would be helpful to have a link for where this binding is defined.

Reply:
Sure, have added a link